plugins/inputs/ping/ping_test.go

```diff
@@ -436,10 +457,9 @@ func TestDNSLookupError(t *testing.T) {
 
 	var acc testutil.Accumulator
 	p := Ping{
-		Urls:        []string{"localhost"},
-		Method:      "native",
-		IPv6:        false,
-		resolveHost: mockHostResolverError,
+		Urls:   []string{"localhost"},
+		Method: "native",
+		IPv6:   false,
```

Comment:
I believe this now does the opposite of what was originally intended! Now the resolve will very likely not fail! Am I missing something?

Reply:
Good catch! Without the `mockHostResolverError` it doesn't make sure an error is thrown. I made a new commit changing "localhost" to "fakehost" that should ensure that the test fails. There shouldn't be a case were someone has a "fakehost"?

Reply:
Yeah but I remember some discussions in the past where someone from InfluxData stated that it's not an option to rely on this. Could you please check back with @ssoroka or @reimda!?

Reply:
yeah I'd rather not depend on a real network address failing, as it fails differently in a lot of various scenarios.

Reply:
I removed the test with the assumption that it is now the 3d party libraries responsibility to detect DNS failures.